aurget

```diff
@@ -158,10 +158,12 @@ sort_rpc() {
   esac
 }
 
+# TODO: SC2076: Don't quote right-hand side of =~, it'll match literally rather than as a regex.
 keep_sources() { [[ " $keep_source_packages " =~ " $1 " ]]; }
```

Comment:
To really resolve this issue, `keep_source_packages` should be an array and we should be checking if `$1` is an element in it here. This has always been a bit of a hack. That said, I'm OK with the hack and would use `shellcheck disable=` to not report on this issue.

Reply:
Okay, I added a comment about the matching, the best I found on StackOverflow. 
Since I did not know what exactly happens there :-) is this okay?